### PR TITLE
fix v0.5.0

### PR DESCRIPTION
**Pull request by jourdain**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jourdain** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,22 +17,22 @@ build:
...
+  number: 1
 
 requirements:
   build:
     - nodejs
   host:
     - hatchling
     - pip
-    - python >=3.7
+    -  python {{ python_min }}
```
Comment (vicentebolea) [inline recipe/meta.yaml:28]: perhaps a >= is missing here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -59,6 +59,7 @@ test:
   requires:
     - pip
     - pytest
+    - python {{ python_min }}
```
Comment (vicentebolea) [inline recipe/meta.yaml:62]: perhaps a >= is missing here

Comment (jourdain) [reply recipe/meta.yaml:62]: no

File: recipe/meta.yaml (hunk 1 above)
Comment (jourdain) [reply recipe/meta.yaml:28]: no

**Pull request merged**